docs/v3/how-to-guides/cloud/connect-to-cloud.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: Connect to Prefect Cloud
+title: How to connect to Prefect Cloud
```

Comment:
these cloud guides need `sidebarTitle`s

Reply:
updated in [3325aaa](https://github.com/PrefectHQ/prefect/pull/18071/commits/3325aaa223423b4fff523f4caf4573542fcd0435)